Libraries/Image/RCTImageView.m

```diff
@@ -151,6 +153,7 @@ - (void)setSource:(RCTImageSource *)source
 {
   if (![source isEqual:_source]) {
     _source = source;
+    [self setDisplayingSource:NO];
```

Comment:
Generally, we prefer to use property access syntax in the RN codebase.

    self.displayingSource = NO;

In this case though, I think it should just be an ivar, avoiding the setter/getter overhead:

    _displayingSource = NO;